bugbug/tools/code_review.py

```diff
@@ -121,6 +123,7 @@ class LargeDiffError(Exception):
 **Valid Comment Examples**:
 
 {comment_examples}
+{approved_examples}
```

Comment:
In the evaluation stage, the good examples could come from the revision itself, so we should prevent that to get accurate evaluation numbers.

Reply:
I filed a follow-up issue: https://github.com/mozilla/bugbug/issues/4995